packages/manager/apps/hub/src/translations/Messages_en_GB.json

```diff
@@ -1,3 +1,4 @@
 {
-  "manager_hub_notification_warning": "Warning: {{ content }}"
+  "manager_hub_notification_warning": "Warning: {{ content }}",
+  "manager_hub_skip_to_main_content": "Skip to main content"
```

Comment:
This functionality is already available across all parts of the Manager so that's why I've added all translations.